lib/cargoAudit.nix

```diff
@@ -1,22 +1,39 @@
-{ cargoBuild
-, cargo-audit
-,
-}: { cargoAuditExtraArgs ? ""
-   , cargoExtraArgs ? ""
-   , advisory-db
-   , ...
-   } @ origArgs:
+{ cargo-audit
+, cargoBuild
+, lib
+}:
+
+{ advisory-db
+, cargoAuditExtraArgs ? ""
+, cargoExtraArgs ? ""
+, src
```

Comment:
I'd be hesitant to change the order of arguments, but it should be safe enough at present.

Reply:
Since we're destructuring the attrset passed in I don't see any issues (Nix doesn't have positional attrset members as far as I know)